demos/durdist2.py

```diff
@@ -9,7 +9,7 @@
 import matplotlib.pyplot as plt
 
 from amads.algorithms import duration_distribution_2
-from amads.io import partitura_midi_import
+from amads.io import music21_midi_import
```

Comment:
Might it be better to use a more generic `amads_midi_import` (or my preferred `import_midi`) in these demos instead?

Reply:
Interesting thoughts! In my experience the `noun_verb` approach often ends up working fine even with OOP, because typically the object (i.e. the instance of the class) is the thing calling the method. e.g.

```py
reader = MIDIReader()
reader.read_midi("file.mid")
```